---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "A simplified approach using Taqman low-density array for medulloblastoma subgrouping"
authors: [Cruzeiro GAV, Salomão KB, de Biagi CAO Jr, Baumgartner M, Sturm D, Lira RCP, de Almeida Magalhães T, Baroni Milan M, da Silva Silveira V, Saggioro FP, de Oliveira RS, Dos Santos Klinger PH, Seidinger AL, Yunes JA, de Paula Queiroz RG, Oba-Shinjo SM, Scrideli CA, Nagahashi SMK, Tone LG, Valera ET]
date: 2019-04-04
doi: "10.1186/s40478-019-0681-y"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: "Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
