<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Carlos Biagi Jr">

  
  
  
    
  
  <meta name="description" content="Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making.">

  
  <link rel="alternate" hreflang="en-us" href="https://cbiagii.github.io/AdipoSNAP/publication/medulloblastoma/">

  


  
  
  
  <meta name="theme-color" content="#1EBA9D">
  

  
  

  
  
  
  
    
    
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light" disabled>
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark">
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=B612+Mono:400,700%7CMerriweather:400,700%7CLato:400,700&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/AdipoSNAP/css/academic.css">

  




  


  
  

  

  <link rel="manifest" href="/AdipoSNAP/index.webmanifest">
  <link rel="icon" type="image/png" href="/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://cbiagii.github.io/AdipoSNAP/publication/medulloblastoma/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="AdipoSNAP">
  <meta property="og:url" content="https://cbiagii.github.io/AdipoSNAP/publication/medulloblastoma/">
  <meta property="og:title" content="A simplified approach using Taqman low-density array for medulloblastoma subgrouping | AdipoSNAP">
  <meta property="og:description" content="Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making."><meta property="og:image" content="https://cbiagii.github.io/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://cbiagii.github.io/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2019-04-04T00:00:00&#43;00:00">
    
    <meta property="article:modified_time" content="2019-04-04T00:00:00&#43;00:00">
  

  


    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://cbiagii.github.io/AdipoSNAP/publication/medulloblastoma/"
  },
  "headline": "A simplified approach using Taqman low-density array for medulloblastoma subgrouping",
  
  "datePublished": "2019-04-04T00:00:00Z",
  "dateModified": "2019-04-04T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "Cruzeiro GAV"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "AdipoSNAP",
    "logo": {
      "@type": "ImageObject",
      "url": "https://cbiagii.github.io/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_192x192_fill_lanczos_center_2.png"
    }
  },
  "description": "Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making."
}
</script>

  

  


  


  





  <title>A simplified approach using Taqman low-density array for medulloblastoma subgrouping | AdipoSNAP</title>

</head>
<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="dark">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  









<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/AdipoSNAP/">AdipoSNAP</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/AdipoSNAP/">AdipoSNAP</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/AdipoSNAP/post/"><span>Tutorials</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  <div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>A simplified approach using Taqman low-density array for medulloblastoma subgrouping</h1>

  

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Cruzeiro GAV</span>, <span >Salomão KB</span>, <span >de Biagi CAO Jr</span>, <span >Baumgartner M</span>, <span >Sturm D</span>, <span >Lira RCP</span>, <span >de Almeida Magalhães T</span>, <span >Baroni Milan M</span>, <span >da Silva Silveira V</span>, <span >Saggioro FP</span>, <span >de Oliveira RS</span>, <span >Dos Santos Klinger PH</span>, <span >Seidinger AL</span>, <span >Yunes JA</span>, <span >de Paula Queiroz RG</span>, <span >Oba-Shinjo SM</span>, <span >Scrideli CA</span>, <span >Nagahashi SMK</span>, <span >Tone LG</span>, <span >Valera ET</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2019
  </span>
  

  

  

  
  
  

  
  

</div>

    













<div class="btn-links mb-3">
  
  








  

















<a class="btn btn-outline-primary my-1 mr-1" href="https://doi.org/10.1186/s40478-019-0681-y" target="_blank" rel="noopener">
  DOI
</a>



</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            
            <a href="/AdipoSNAP/publication/#2">
              Journal article
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    

    <div class="space-below"></div>

    <div class="article-style"></div>

    






















  
  
  <div class="article-widget content-widget-hr">
    <h3>Related</h3>
    <ul>
      
      <li><a href="/AdipoSNAP/publication/ultraconserved/">Ultraconserved long non-coding RNA uc.112 is highly expressed in childhood T versus B-cell acute lymphoblastic leukemia</a></li>
      
      <li><a href="/AdipoSNAP/publication/zeb1/">Whole transcriptome analysis reveals correlation of long noncoding RNA ZEB1-AS1 with invasive profile in melanoma</a></li>
      
    </ul>
  </div>
  





  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/AdipoSNAP/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/AdipoSNAP/js/academic.min.66c553246b0f279a03be6e5597f72b52.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    
  </p>

  
  






  <p class="powered-by">
    
    Published with
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic Website Builder</a>
    

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
