<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Carlos Biagi Jr">

  
  
  
    
  
  <meta name="description" content="">

  
  <link rel="alternate" hreflang="en-us" href="https://cbiagii.github.io/AdipoSNAP/publication/">

  


  
  
  
  <meta name="theme-color" content="#1EBA9D">
  

  
  

  
  
  
  
    
    
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css" integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css" crossorigin="anonymous" title="hl-light" disabled>
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark">
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      
        
      

      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=B612+Mono:400,700%7CMerriweather:400,700%7CLato:400,700&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/AdipoSNAP/css/academic.css">

  




  


  
  

  
  <link rel="alternate" href="/AdipoSNAP/publication/index.xml" type="application/rss+xml" title="AdipoSNAP">
  

  <link rel="manifest" href="/AdipoSNAP/index.webmanifest">
  <link rel="icon" type="image/png" href="/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://cbiagii.github.io/AdipoSNAP/publication/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="og:site_name" content="AdipoSNAP">
  <meta property="og:url" content="https://cbiagii.github.io/AdipoSNAP/publication/">
  <meta property="og:title" content="Publications | AdipoSNAP">
  <meta property="og:description" content=""><meta property="og:image" content="https://cbiagii.github.io/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://cbiagii.github.io/AdipoSNAP/images/icon_hu64d5fdaaa5c80254dbe3c681e74a1b81_409389_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2020-08-20T00:00:00&#43;00:00">
  

  




  


  





  <title>Publications | AdipoSNAP</title>

</head>
<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="dark">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  









<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/AdipoSNAP/">AdipoSNAP</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/AdipoSNAP/">AdipoSNAP</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/AdipoSNAP/post/"><span>Tutorials</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>


  












  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Publications</h1>

  

  
</div>



<div class="universal-wrapper">
  <div class="row">
    <div class="col-lg-12">

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <div class="form-row mb-4">
        <div class="col-auto">
          <input type="search" class="filter-search" placeholder="Search..." autocapitalize="off"
          autocomplete="off" autocorrect="off" role="textbox" spellcheck="false">
        </div>
        <div class="col-auto">
          <select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group="pubtype">
            <option value="*">Type</option>
            
            
            <option value=".pubtype-2">
              Journal article
            </option>
            
            <option value=".pubtype-3">
              Preprint
            </option>
            
          </select>
        </div>
        <div class="col-auto">
          <select class="pub-filters form-control form-control-sm" data-filter-group="year">
            <option value="*">Date</option>
            
            
            
            <option value=".year-2020">
              2020
            </option>
            
            <option value=".year-2019">
              2019
            </option>
            
            <option value=".year-2018">
              2018
            </option>
            
            
          </select>
        </div>
      </div>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Ferreira MR; Santos GA; Biagi CA; Silva Junior</span>, <span >WA; Zambuzzi WF</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/gsva/">GSVA score reveals molecular signatures from transcriptomes for biomaterials comparison</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/gsva/" class="summary-link">
    <div class="article-style">
      <p>Two in silico methodologies were implemented to reveal the molecular signatures of inorganic Hydroxyapatite (HA) and ß-TCP materials from a transcriptome database to compare biomaterials. To test this new methodology, we choose the array E-MTAB-7219, which contains the transcription profile of osteoblastic cell line seeded onto 15 different biomaterials up to 48 h. The expansive potential of the methodology was tested from the construction of customized signatures. We present, for the first time, a methodology to compare the performance of different biomaterials using the transcriptome profile of the cell through the GSVA score. To test this methodology, we implemented two methods based on MSigDB collections, using all the collections and sub-collections except the Hallmark collection, which was used in the second method. The result of this analysis provided an initial understanding of biomaterial grouping based on the cell transcriptional landscape. The comparison using GSVA score combined efforts and expand the potential to compare biomaterials using transcriptome profile. Altogether, our results provide a better understanding of the comparison of different biomaterials and suggest a possibility of the new methodology be applied to the prospection of new biomaterials.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1002/jbm.a.37090" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Codo AC</span>, <span >Davanzo GG</span>, <span >Monteiro LB</span>, <span >de Souza GF</span>, <span >Muraro SP</span>, <span >Virgilio-da-Silva JV</span>, <span >Prodonoff JS</span>, <span >Carregari VC</span>, <span >de Biagi Junior CAO</span>, <span >Crunfli F</span>, <span >Jimenez Restrepo JL</span>, <span >Vendramini PH</span>, <span >Reis-de-Oliveira G</span>, <span >Bispo Dos Santos K</span>, <span >Toledo-Teixeira DA</span>, <span >Parise PL</span>, <span >Martini MC</span>, <span >Marques RE</span>, <span >Carmo HR</span>, <span >Borin A</span>, <span >Coimbra LD</span>, <span >Boldrini VO</span>, <span >Brunetti NS</span>, <span >Vieira AS</span>, <span >Mansour E</span>, <span >Ulaf RG</span>, <span >Bernardes AF</span>, <span >Nunes TA</span>, <span >Ribeiro LC</span>, <span >Palma AC</span>, <span >Agrela MV</span>, <span >Moretti ML</span>, <span >Sposito AC</span>, <span >Pereira FB</span>, <span >Velloso LA</span>, <span >Vinolo MAR</span>, <span >Damasio A</span>, <span >Proença-Módena JL</span>, <span >Carvalho RF</span>, <span >Mori MA</span>, <span >Martins-de-Souza D</span>, <span >Nakaya HI</span>, <span >Farias AS</span>, <span >Moraes-Vieira PM</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/covid/">Elevated Glucose Levels Favor SARS-CoV-2 Infection and Monocyte Response through a HIF-1α/Glycolysis-Dependent Axis</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/covid/" class="summary-link">
    <div class="article-style">
      <p>COVID-19 can result in severe lung injury. It remained to be determined why diabetic individuals with uncontrolled glucose levels are more prone to develop the severe form of COVID-19. The molecular mechanism underlying SARS-CoV-2 infection and what determines the onset of the cytokine storm found in severe COVID-19 patients are unknown. Monocytes and macrophages are the most enriched immune cell types in the lungs of COVID-19 patients and appear to have a central role in the pathogenicity of the disease. These cells adapt their metabolism upon infection and become highly glycolytic, which facilitates SARS-CoV-2 replication. The infection triggers mitochondrial ROS production, which induces stabilization of hypoxia-inducible factor-1α (HIF-1α) and consequently promotes glycolysis. HIF-1α-induced changes in monocyte metabolism by SARS-CoV-2 infection directly inhibit T cell response and reduce epithelial cell survival. Targeting HIF-1ɑ may have great therapeutic potential for the development of novel drugs to treat COVID-19.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.cmet.2020.07.007" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Carlos Alberto Oliveira de Biagi Jr</span>, <span >Ricardo Perecin Nociti</span>, <span >Breno Osvaldo Funicheli</span>, <span >Patrícia de Cássia Ruy</span>, <span >João Paulo Bianchi Ximenez</span>, <span >Wilson Araújo Silva Jr</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/cetf/">CeTF: an R package to Coexpression for Transcription Factors using Regulatory Impact Factors (RIF) and Partial Correlation and Information (PCIT) analysis</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/cetf/" class="summary-link">
    <div class="article-style">
      <p>Finding meaningful gene-gene associations and the main Transcription Factors (TFs) in co-expression networks is one of the most important challenges in gene expression data mining. CeTF is an R package that integrates the Partial Correlation with Information Theory (PCIT) and Regulatory Impact Factors (RIF) algorithms applied to gene expression data from microarray, RNA-seq, or single-cell RNA-seq platforms. This approach allows identifying the transcription factors most likely to regulate a given network in different biological systems — for example, regulation of gene pathways in tumor stromal cells and tumor cells of the same tumor. This pipeline can be easily integrated into the high-throughput analysis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.03.30.015784" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >das Chagas PF</span>, <span >de Sousa GR</span>, <span >Kodama MH</span>, <span >de Biagi Junior CAO</span>, <span >Yunes JA</span>, <span >Brandalise SR</span>, <span >Calin GA</span>, <span >Tone LG</span>, <span >Scrideli CA</span>, <span >de Oliveira JC</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/ultraconserved/">Ultraconserved long non-coding RNA uc.112 is highly expressed in childhood T versus B-cell acute lymphoblastic leukemia</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/ultraconserved/" class="summary-link">
    <div class="article-style">
      <p>Aberrant expression of long non-coding RNAs (lncRNAs) has been detected in several types of cancer, including acute lymphoblastic leukemia (ALL), but lncRNA mapped on transcribed ultraconserved regions (T-UCRs) are little explored. The T-UCRs uc.112, uc.122, uc.160 and uc.262 were evaluated by quantitative real-time PCR in bone marrow samples from children with T-ALL (n=32) and common-ALL/pre-B ALL (n=30). In pediatric ALL, higher expression levels of uc.112 were found in patients with T-ALL, compared to patients with B-ALL. T-cells did not differ significantly from B-cells regarding uc.112 expression in non-tumor precursors from public data. Additionally, among B-ALL patients, uc.112 was also found to be increased in patients with hyperdiploidy, compared to other karyotype results. The uc.122, uc.160, and uc.262 were not associated with biological or clinical features. These findings suggest a potential role of uc.112 in pediatric ALL and emphasize the need for further investigation of T-UCR in pediatric ALL.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.htct.2019.12.003" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Siena ÁDD</span>, <span >Plaça JR</span>, <span >Araújo LF</span>, <span >de Barros II</span>, <span >Peronni K</span>, <span >Molfetta G</span>, <span >de Biagi CAO Jr</span>, <span >Espreafico EM</span>, <span >Sousa JF</span>, <span >Silva WA Jr</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/zeb1/">Whole transcriptome analysis reveals correlation of long noncoding RNA ZEB1-AS1 with invasive profile in melanoma</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/zeb1/" class="summary-link">
    <div class="article-style">
      <p>Melanoma is the deadliest form of skin cancer, and little is known about the impact of deregulated expression of long noncoding RNAs (lncRNAs) in the progression of this cancer. In this study, we explored RNA-Seq data to search for lncRNAs associated with melanoma progression. We found distinct lncRNA gene expression patterns across melanocytes, primary and metastatic melanoma cells. Also, we observed upregulation of the lncRNA ZEB1-AS1 (ZEB1 antisense RNA 1) in melanoma cell lines. Data analysis from The Cancer Genome Atlas (TCGA) confirmed higher ZEB1-AS1 expression in metastatic melanoma and its association with hotspot mutations in BRAF (B-Raf proto-oncogene, serine/threonine kinase) gene and RAS family genes. In addition, a positive correlation between ZEB1-AS1 and ZEB1 (zinc finger E-box binding homeobox 1) gene expression was verified in primary and metastatic melanomas. Using gene expression signatures indicative of invasive or proliferative phenotypes, we found an association between ZEB1-AS1 upregulation and a transcriptional profile for invasiveness. Enrichment analysis of correlated genes demonstrated cancer genes and pathways associated with ZEB1-AS1. We suggest that the lncRNA ZEB1-AS1 could function by activating ZEB1 gene expression, thereby influencing invasiveness and phenotype switching in melanoma, an epithelial-to-mesenchymal transition (EMT)-like process, which the ZEB1 gene has an essential role.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41598-019-47363-6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Cruzeiro GAV</span>, <span >Salomão KB</span>, <span >de Biagi CAO Jr</span>, <span >Baumgartner M</span>, <span >Sturm D</span>, <span >Lira RCP</span>, <span >de Almeida Magalhães T</span>, <span >Baroni Milan M</span>, <span >da Silva Silveira V</span>, <span >Saggioro FP</span>, <span >de Oliveira RS</span>, <span >Dos Santos Klinger PH</span>, <span >Seidinger AL</span>, <span >Yunes JA</span>, <span >de Paula Queiroz RG</span>, <span >Oba-Shinjo SM</span>, <span >Scrideli CA</span>, <span >Nagahashi SMK</span>, <span >Tone LG</span>, <span >Valera ET</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/medulloblastoma/">A simplified approach using Taqman low-density array for medulloblastoma subgrouping</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/medulloblastoma/" class="summary-link">
    <div class="article-style">
      <p>Next-generation sequencing platforms are routinely used for molecular assignment due to their high impact for risk stratification and prognosis in medulloblastomas. Yet, low and middle-income countries still lack an accurate cost-effective platform to perform this allocation. TaqMan Low Density array (TLDA) assay was performed using a set of 20 genes in 92 medulloblastoma samples. The same methodology was assessed in silico using microarray data for 763 medulloblastoma samples from the GSE85217 study, which performed MB classification by a robust integrative method (Transcriptional, Methylation and cytogenetic profile). Furthermore, we validated in 11 MBs samples our proposed method by Methylation Array 450 K to assess methylation profile along with 390 MB samples (GSE109381) and copy number variations. TLDA with only 20 genes accurately assigned MB samples into WNT, SHH, Group 3 and Group 4 using Pearson distance with the average-linkage algorithm and showed concordance with molecular assignment provided by Methylation Array 450 k. Similarly, we tested this simplified set of gene signatures in 763 MB samples and we were able to recapitulate molecular assignment with an accuracy of 99.1% (SHH), 94.29% (WNT), 92.36% (Group 3) and 95.40% (Group 4), against 97.31, 97.14, 88.89 and 97.24% (respectively) with the Ward.D2 algorithm. t-SNE analysis revealed a high level of concordance (k = 4) with minor overlapping features between Group 3 and Group 4. Finally, we condensed the number of genes to 6 without significantly losing accuracy in classifying samples into SHH, WNT and non-SHH/non-WNT subgroups. Additionally, we found a relatively high frequency of WNT subgroup in our cohort, which requires further epidemiological studies. TLDA is a rapid, simple and cost-effective assay for classifying MB in low/middle income countries. A simplified method using six genes and restricting the final stratification into SHH, WNT and non-SHH/non-WNT appears to be a very interesting approach for rapid clinical decision-making.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s40478-019-0681-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Araujo LF</span>, <span >Siena ADD</span>, <span >Plaça JR</span>, <span >Brotto DB</span>, <span >Barros II</span>, <span >Muys BR</span>, <span >Biagi CAO Jr</span>, <span >Peronni KC</span>, <span >Sousa JF</span>, <span >Molfetta GA</span>, <span >West LC</span>, <span >West AP</span>, <span >Leopoldino AM</span>, <span >Espreafico EM</span>, <span >Silva WA Jr</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/AdipoSNAP/publication/tfam/">Mitochondrial transcription factor A (TFAM) shapes metabolic and invasion gene signatures in melanoma</a>
  </h3>

  
  <a href="/AdipoSNAP/publication/tfam/" class="summary-link">
    <div class="article-style">
      <p>Mitochondria are central key players in cell metabolism, and mitochondrial DNA (mtDNA) instability has been linked to metabolic changes that contribute to tumorigenesis and to increased expression of pro-tumorigenic genes. Here, we use melanoma cell lines and metastatic melanoma tumors to evaluate the effect of mtDNA alterations and the expression of the mtDNA packaging factor, TFAM, on energetic metabolism and pro-tumorigenic nuclear gene expression changes. We report a positive correlation between mtDNA copy number, glucose consumption, and ATP production in melanoma cell lines. Gene expression analysis reveals a down-regulation of glycolytic enzymes in cell lines and an up-regulation of amino acid metabolism enzymes in melanoma tumors, suggesting that TFAM may shift melanoma fuel utilization from glycolysis towards amino acid metabolism, especially glutamine. Indeed, proliferation assays reveal that TFAM-down melanoma cell lines display a growth arrest in glutamine-free media, emphasizing that these cells rely more on glutamine metabolism than glycolysis. Finally, our data indicate that TFAM correlates to VEGF expression and may contribute to tumorigenesis by triggering a more invasive gene expression signature. Our findings contribute to the understanding of how TFAM affects melanoma cell metabolism, and they provide new insight into the mechanisms by which TFAM and mtDNA copy number influence melanoma tumorigenesis.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  

















<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41598-018-31170-6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js" integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = true;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/AdipoSNAP/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/AdipoSNAP/js/academic.min.66c553246b0f279a03be6e5597f72b52.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    
  </p>

  
  






  <p class="powered-by">
    
    Published with
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic Website Builder</a>
    

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
